src/node_crypto.cc

```diff
@@ -5616,24 +5616,20 @@ void RandomBytesBuffer(const FunctionCallbackInfo<Value>& args) {
 
 void GetSSLCiphers(const FunctionCallbackInfo<Value>& args) {
   Environment* env = Environment::GetCurrent(args);
+  Isolate* isolate = args.GetIsolate();
 
   SSL_CTX* ctx = SSL_CTX_new(TLSv1_server_method());
-  if (ctx == nullptr) {
-    return env->ThrowError("SSL_CTX_new() failed.");
-  }
+  CHECK_NE(ctx, nullptr);  // SSL_CTX_new failed
 
   SSL* ssl = SSL_new(ctx);
-  if (ssl == nullptr) {
-    SSL_CTX_free(ctx);
-    return env->ThrowError("SSL_new() failed.");
-  }
+  CHECK_NE(ssl, nullptr);  // SS_new failed
 
   Local<Array> arr = Array::New(env->isolate());
   STACK_OF(SSL_CIPHER)* ciphers = SSL_get_ciphers(ssl);
 
   for (int i = 0; i < sk_SSL_CIPHER_num(ciphers); ++i) {
     const SSL_CIPHER* cipher = sk_SSL_CIPHER_value(ciphers, i);
-    arr->Set(i, OneByteString(args.GetIsolate(), SSL_CIPHER_get_name(cipher)));
+    arr->Set(i, OneByteString(isolate, SSL_CIPHER_get_name(cipher)));
```

Comment:
This change seems unrelated.

Reply:
Unrelated, yes, but while I was in here I figured I'd clean up a bit.

Reply:
A commit should not contain unrelated cleanup.  It should do what it says on the tin, no more and no less.

Reply:
Will update the commit message to include the additional change

Reply:
Ah, that's what I get for using pithy one-liners.  What I mean is, cleanup is fine but do it in a separate commit.  Future code archeologists will thank you for it.

Having said that, this specific cleanup doesn't buy much.  It's neither shorter nor faster.